### ISSUE TITLE
Treegrid example does not expand/collapse on default action

### ISSUE THREAD

**Issue by dlibby-** [w3c/aria-practices#1849]
https://www.w3.org/TR/wai-aria-practices-1.1/examples/treegrid/treegrid-1.html

Not sure whether this is in scope (after all the example seems to be geared towards keyboard usage), but we've seen web developers copy this pattern and leave screen reader users unable to easily expand/collapse the items without changing modes to get the keyboard mappings as defined by the treegrid.

The table rows have `aria-expanded` defined on them, but the click handler (invoked via AT's default action) uses some coordinate detection to determine whether or not the click was over the triangle. Note that doubleclick handler _does_ perform expand/collapse. Per [CoreAAM](https://w3c.github.io/core-aam/#mapping_actions), all default actions map to clicks so this pattern seems problematic.

Let me know if any of this is unclear or I'm missing anything - this is my first issue I've filed on this repo.

**Comment by dlibby-** [comment]
@mcking65 - any thoughts on this issue? Thanks!
